plugin-helpers.gradle

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2017 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import java.text.DecimalFormat
+
+static String gitRevision() {
+    def process = "git log -n 1 --format=%H".execute()
+    process.waitFor()
+    return process.text.stripIndent().trim()
+}
+
+static String distVersion() {
+    def process = "git rev-list --count HEAD".execute()
```

Comment:
`--count` is not supported by the older version of the git.

Reply:
Corrected.